### PR TITLE
WIP: SRT changes 

### PR DESCRIPTION
**Pull request by kierank**

### CODE REVIEW

**Review by funman — APPROVED** [review]
Looks good, is it ready?

File: lib/upipe-srt/upipe_srt_receiver.c [hunk 1]
```diff
@@ -617,11 +622,19 @@ static void upipe_srt_receiver_timer(struct upump *upump)
...
 
-        if (--upipe_srt_receiver->buffered == 0) {
-            upipe_warn_va(upipe, "Exhausted buffer");
-            upipe_srt_receiver->expected_seqnum = UINT64_MAX;
-            upipe_srt_receiver->last_output_seqnum = UINT64_MAX;
-        }
+        upipe_srt_receiver->buffered--;
+    }
+
+    /* At low bitrates and VBR video the buffer may momentarily exhaust.
```
Comment (funman) [inline lib/upipe-srt/upipe_srt_receiver.c]: Whitespace at the end of line triggers coding style buildbot

Comment (kierank) [reply lib/upipe-srt/upipe_srt_receiver.c]: I will remove this commit

**Comment by kierank** [comment]
This should be RTT/2

**Comment by kierank** [comment]
Causes problems if the latency is set to less than the initial RTT estimator of 100ms

**Pull request closed without merging**